chia/wallet/wallet_transaction_store.py

```diff
@@ -52,25 +52,23 @@ async def create(cls, db_wrapper: DBWrapper):
             "CREATE INDEX IF NOT EXISTS tx_confirmed_index on transaction_record(confirmed_at_height)"
         )
 
-        await self.db_connection.execute(
-            "CREATE INDEX IF NOT EXISTS tx_created_index on transaction_record(created_at_time)"
-        )
-
+        # index tx_created_index is a duplicate to tx_created_time and can be dropped
+        await self.db_connection.execute("DROP INDEX IF EXISTS tx_created_index on transaction_record(created_at_time)")
         await self.db_connection.execute("CREATE INDEX IF NOT EXISTS tx_confirmed on transaction_record(confirmed)")
-
-        await self.db_connection.execute("CREATE INDEX IF NOT EXISTS tx_sent on transaction_record(sent)")
-
-        await self.db_connection.execute(
-            "CREATE INDEX IF NOT EXISTS tx_created_time on transaction_record(created_at_time)"
-        )
+        # this index is not used by any queries, don't create it for new
+        # installs, and remove it from existing installs in the future
+        # await self.db.execute("DROP INDEX IF EXISTS tx_sent on transaction_record(sent)")
+        # this index is not used by any queries, don't create it for new
+        # installs, and remove it from existing installs in the future
+        # await self.db.execute("DROP INDEX IF EXISTS tx_created_time on transaction_record(created_at_time)")
 
         await self.db_connection.execute("CREATE INDEX IF NOT EXISTS tx_type on transaction_record(type)")
 
-        await self.db_connection.execute(
-            "CREATE INDEX IF NOT EXISTS tx_to_puzzle_hash on transaction_record(to_puzzle_hash)"
-        )
+        # this index is not used by any queries, don't create it for new
+        # installs, and remove it from existing installs in the future
+        # await self.db.execute("DROP INDEX IF EXISTS tx_to_puzzle_hash on transaction_record(to_puzzle_hash)")
 
-        await self.db_connection.execute("CREATE INDEX IF NOT EXISTS wallet_id on transaction_record(wallet_id)")
+        await self.db_connection.execute("CREATE INDEX IF NOT EXISTS tx_wallet_id on transaction_record(wallet_id)")
```

Comment:
why do you rename this index? It would make everyone upgrading spending time creating a new (identical) index.

Reply:
Similar to other indexes I renamed. On my wallet dbs there is currently no index on transaction_record table, as they are conflicting by name. index named "wallet_id" for example already exists on coin_record table.